src/components/home/home.tsx

```diff
@@ -68,18 +68,20 @@ export const Home = ({
               display={{ base: "none", lg: "block" }}
             >
               <RandomFeaturedProject metrics={sort} />
-              <SectionHeading
-                icon={<GoTag fontSize={32} />}
-                title="Popular Tags"
-              />
-              <CompactTagList
-                tags={popularTags}
-                footer={
-                  <Button as={RouterLink} to={`/tags/`} variant="link">
-                    View all tags »
-                  </Button>
-                }
-              />
+              <div data-testid="popular-tags-section-desktop">
```

Comment:
Are we are changing the DOM structure, adding a new `<div>` only for testing purpose?

Besides the fact that adding an extra div might have CSS effects, usually I try to remove `div` that are not necessary because every DOM element has a cost and it makes inspecting things in DevTools more complicated.

However we may need a wrapper for these 2 component that are showed/hidden together depending on the screen size, we may need some refactoring, so your approach may be correct too, I need to think more...

Reply:
> Are we are changing the DOM structure, adding a new \<div\> only for testing purpose?

Yes. I need to change DOM so I can easily query (2).
<img width="324" alt="popular tags in homepage" src="https://user-images.githubusercontent.com/8603085/170609180-afa6ba3b-0acd-4a23-b8a0-258c838877bb.png">

Besides, I strongly think (1) and (2) should be inside a wrapper, I believe that would be a better HTML structure if `Featured Projects` and `Popular Tags` are on the same HTML level. (Now `Featured Projects`, the heading `Popular Tags` and the list are on the same level)

<img width="553" alt="side bar code" src="https://user-images.githubusercontent.com/8603085/170610198-3093c381-9a79-4302-8b5c-9050f46116c6.png">

https://github.com/bestofjs/bestofjs-webui/blob/bb662931fe8e965e74575ebb036727436f85cd2f/src/components/home/home.tsx#L70-L82

> Besides the fact that adding an extra div might have CSS effects

I did think about if it might affect CSS. I tested it myself and look like it's going to be fine. Please help to test it by yourself.

> usually I try to remove div that are not necessary because every DOM element has a cost and it makes inspecting things in DevTools more complicated.

I 100% agree, but without changing the DOM, I couldn't test the `Popular Tags` part. So I guess this is the change we need to make. Please propose any other suggestion, since you are more familiar with Best Of JS code base.